tests/components/tado/util.py

```diff
@@ -165,3 +165,8 @@ async def async_init_integration(
         if not skip_setup:
             await hass.config_entries.async_setup(entry.entry_id)
             await hass.async_block_till_done()
+
+
+def dummy_tado_connector(hass: HomeAssistant, fallback) -> HomeAssistant:
```

Comment:
Why create a dummy and not patch the parts in the tests itself?

Reply:
mainly because part that I am testing can just use the original object. I just wanted to do unit testing on the new decide_overlay_mode method